fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -1333,6 +1333,30 @@ public class Config extends ConfigBase {
     @ConfField(mutable = true)
     public static long memory_tracker_interval_seconds = 60;
 
+    /**
+     * true to enable collect proc memory alloc profile
+     */
+    @ConfField(mutable = true)
+    public static boolean proc_profile_enable = true;
+
+    /**
+     * The number of seconds between proc profile collections
```

Comment:
please write it in the `comment` of `ConfField`

Reply:
done